vue/components/ui/molecules/calendar/calendar.vue

```diff
@@ -147,7 +147,7 @@ export const Calendar = {
          * Example: "2020/12/31"
          */
         value: {
-            type: String,
+            type: String | Number,

```

Comment:
Why should we support string?

Reply:
I figured it could simplify extra formatting effort in case the server outputs an epoch in string format.
I can revert

Reply:
yea please revert, keeping interfaces as thin as possible makes them easier to maintain.

Reply:
nvm, i know you already did